tutor/src/components/reference-book/page.cjsx

```diff
@@ -71,6 +75,7 @@ module.exports = React.createClass
       title: @getSplashTitle()
 
     <div className={classnames('page-wrapper', @props.className)}>
+      <tutor-highlight></tutor-highlight>
       {@props.children}
```

Comment:
Hm, rather than making up our own DOM node types can we use a `<div data-type="tutor-highlight" />`

Maybe this is here by mistake though?  It seems like something that `AnnotatorWidget` could add when it renders.

Reply:
That particular tag shouldn't be there at all.